projects/ad400x-st/README.rst

```diff
@@ -0,0 +1,104 @@
+Evaluating AD400x on SDP-K1
+===========================
+
+Contents
+--------
+
+.. contents:: Table of Contents
+    :depth: 3
+
+Supported Evaluation Boards
+---------------------------
+
+* `AD400x <https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/EVAL-AD400x-FMCZ.html>`_
+
+Overview
+--------
+
+Features and Benefits
+
+Product Details
+The AD4000/AD4001/AD4002/AD4003/AD4020 family of evaluation boards enable quick,
+simplified evaluation of the AD4000 family of 16-/18-/20-bit, precision
+successive approx-imation register (SAR) analog-to-digital converters (ADCs).
+
+The AD4000/AD4001/AD4002/AD4003/AD4020 are low power, 16-bit/18-bit/20-bit,
+precision SAR ADCs that offer high performance with throughputs up to 2 MSPS
+(1.8 MSPS for the AD4020). The evaluation board demonstrates the performance
+of the AD4000/AD4001/AD4002/AD4003/AD4020 family of ADCs and provides a
+simplified interface for a variety of system applications.
+
+Applications
+------------
+
+* Industrial Automation Technology
+* Instrumentation and Measurement Solutions
+* Intelligent Building Solutions
+* Healthcare Solutions
+* Aerospace and Defense Systems
+* Wireless Communication Solutions
+
+Hardware Specifications
+-----------------------
+
```

Comment:
nothing added in this section, please remove it if it is not needed

Reply:
ok